MessengerProj/src/main/res/values/strings.xml

```diff
@@ -10,21 +10,21 @@
     <string name="NoResult">No results.</string>
     <string name="NoChats">No chats yet.</string>
     <string name="DeleteChat">Delete chat</string>
-    <string name="SelectChat">Select chat …</string>
+    <string name="SelectChat">Select chat…</string>
```

Comment:
Well, it's not clear if the ellipsis is surrounded by space or not, see https://en.wikipedia.org/wiki/Ellipsis , eg. in menus, Ubuntu adds a space and Firefox does not.
I personally prefer the space, as without a space, to me it always suggests, the the _word_ is not complete. The space suggests, the _sentence_ is not complete. In our scenario, it is the action that is not complete, but I think this is closes to the latter one.

Reply:
I prefer to have the space as well.